### PR TITLE
Travis CI: Write & print DIRECTORY.md on one line

### PR DESCRIPTION
**Pull request by mrvnmchm**
I have been successful at updating the DIRECTORY.md file. However, after multiple tests and failures I am unable to achieve pushing back to the PR that initiated the build. Apparently this is due to Travis CI not including secure environment variables in PR builds, see travis-ci/travis-ci#1500

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: python
 python: 3.7
 cache: pip
-before_install: pip install --upgrade pip setuptools
+before_install: 
+  - pip install --upgrade pip setuptools
```
Comment (cclauss) [inline .travis.yml]: Why this change?  How is it different than the current code?

File: .travis.yml [hunk 2]
```diff
@@ -11,5 +12,4 @@ script:
   - mypy --ignore-missing-imports .
   - pytest . --doctest-modules
 after_success:
-  - scripts/build_directory_md.py > DIRECTORY.md
-  - cat DIRECTORY.md
+  - scripts/build_directory_md.py 2>&1 | tee DIRECTORY.md
```
Comment (cclauss) [inline .travis.yml]: Why this change?  How is it different than the current code?

File: .travis.yml (hunk 1 above)
Comment (mrvnmchm) [reply .travis.yml]: Formatting leftovers during previous attempts of testing builds. I'll revert this.

File: .travis.yml (hunk 2 above)
Comment (mrvnmchm) [reply .travis.yml]: This change eliminates a line and does both operations. Printing the output, and writing it to the file, see [tee](https://en.wikipedia.org/wiki/Tee_(command))

**Comment by chttrjeankr** [comment]
I see the Travis CI build didn't pass. Isn't that a problem?

**Comment by cclauss** [comment]
I made a whitespace change to @mrvnmchm’s PR before landing it which confuses Travis CI but all is well.  https://travis-ci.com/TheAlgorithms/Python/builds/134508235#L170
